ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/ReasonForDivorce.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.divorce.ccd.model.enums;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum ReasonForDivorce implements HasLabel {
```

Comment:
We will no longer as this so it should be removed

Reply:
Will remove this